src/core/index.md

```diff
@@ -1,63 +1,61 @@
 # Core
 
-DataJoint Core projects are fully open-source and are built to develop, define, manage,
-and visualize [data pipelines](./getting-started/data-pipelines). Below are the projects that
-make up the family of core open-source projects.
+DataJoint Core is a collection of open-source tools for building and operating scientific data pipelines.
```

Comment:
For the sake of SEO, we should make sure to link back to glossary and between pages as much as possible.

```suggestion
DataJoint Core is a collection of open-source tools for building, operating, and visualizing scientific [data pipelines](./concepts/glossary#data-pipeline).
```

Reply:
TBD in subsequent PR's.